### PR TITLE
Add the e3nn model with the new pipeline

### PR DESCRIPTION
**Pull request by Adam-Mazur**

### CODE REVIEW

File: src/models/e3nn/model.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+        # Global pooling (rotation invariant)
+        x_max = torch.max(x, dim=1)[0]
+        x_mean = torch.mean(x, dim=1)
+        x_std = torch.std(x, dim=1)
+
+        # Concatenate all pooling strategies
+        x = torch.cat([x_max, x_mean, x_std], dim=-1)
+
+        x = torch.relu(self.fc1(self.dropout(x)))
```
Comment (rybijakKarpiowy) [inline src/models/e3nn/model.py]: You could put those three lines into a self.ClassificationHead for clarity. Why don't we use activation function at the end? Softmax or smth
